src/View/AdminUser/InitComponent.java

```diff
@@ -0,0 +1,65 @@
+package View.AdminUser;
+
+import javax.swing.JPanel;
+import javax.swing.JLabel;
+import javax.swing.JTextField;
+import javax.swing.JComboBox;
+import java.awt.Dimension;
+import java.awt.Component;
+
+/**
+ * Class to help us to add components like JLabel, JTextBox, JComboBox
+ *
+ * @author Lourdes Villca
+ */
+public class InitComponent {
+    /**
+     * Method to initialize a JLabel
+     *
+     * @param jlabel component to be created
+     * @param text   Text of the Jlabel
+     * @param panel  Panel where the component will be added
+     */
+    public static void initLabel(JLabel jlabel, String text, JPanel panel) {
```

Comment:
if all the methods have the same paramete,rs you could have a empty classe so represents